### PR TITLE
Don't blame user on TargetAbortedException (fixes bnc#978193)

### PR DESCRIPTION
**Pull request by bzeller**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,4 @@ build
 .externalToolBuilders
 .settings
 *flymake.h.gch
-*.user
+CMakeLists.txt.user*
```
Comment (mlandres) [inline .gitignore]: .gitignore change is unrelated should go into a separate commit.

Comment (bzeller) [reply .gitignore]: Done
